src/openms/include/OpenMS/COMPARISON/SPECTRA/SpectrumAlignment.h

```diff
@@ -51,9 +51,10 @@ namespace OpenMS
   /**
       @brief Aligns the peaks of two spectra
 
-        Using a banded (width via 'tolerance' parameter) alignment.
+        Using a banded (width via 'tolerance' parameter) alignment if absolute tolerances are given.
         Scoring function is the m/z distance between peaks. Intensity does not play a role!
 
+        If relative tolerance (ppm) is given simple matching of peaks is performed.
```

Comment:
ok will do. would be nice if one could (later) also add an option for resolving the double assignment (e.g. only to higher intensity peak) but this would not work with the absolute tolerance alignment